sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java

```diff
@@ -154,6 +157,17 @@ private MongoDbIO() {}
 
     abstract int numSplits();
 
+    abstract int limit();
+
+    @Nullable
+    abstract List<BsonDocument> pipeline();
```

Comment:
Is `pipeline` a standard thing from MongoDb? It can be confusing because Beam also has a totally unrelated pipeline concept.

Reply:
I can rename this to aggregates, both are interchangeable in the MongoDB community.
https://docs.mongodb.com/manual/aggregation/